src/handlers/call-handler.js

```diff
@@ -0,0 +1,39 @@
+import { isFunction } from '../utils/types'
+import * as emitter from '../utils/emitter'
+
+/**
+ * @typedef {Object} CallHandler
+ * @property {function} [ajaxGet]
+ * @property {function} [pixelGet]
+ */
+
+const _noOp = () => undefined
+
+/**
+ * @param {CallHandler} externalCallHandler
+ * @returns {CallHandler}
+ * @constructor
+ */
+export function CallHandler (externalCallHandler) {
+  const errors = []
+
+  function _externalOrError (functionName) {
+    const hasExternal = externalCallHandler && externalCallHandler[functionName] && isFunction(externalCallHandler[functionName])
+    if (hasExternal) {
+      return externalCallHandler[functionName]
+    } else {
+      errors.push(functionName)
```

Comment:
What's the reason for storing errors here? Isn't it better to just emit those and be done with it?

Reply:
I just wanted to have only one error message. 
For example, in the storage we expect 7 functions to be provided. And adding errors into the array prevents us from sending 7 errors: one per each missing function.
The implementation is the same in the `call` handler so that there is only 1 error send instead of 2.

Reply:
Gotcha...